be/test/formats/parquet/arrow_parquet_writer_test.cpp

```diff
@@ -0,0 +1,84 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+#include <parquet/api/writer.h>
+
+#include "util/slice.h"
+
+namespace starrocks::parquet {
+
+class MockOutputStream : public arrow::io::OutputStream {
+public:
+    MockOutputStream() = default;
+
+    ~MockOutputStream() override = default;
+
+    arrow::Status Write(const void* data, int64_t nbytes) override {
+        _counter++;
+        if (_counter == 3) { // return error when flush the 2nd column chunk
+            return arrow::Status::IOError("mock io error");
+        }
+        // std::cout << "write " << _counter << " times data = " << Slice(reinterpret_cast<const uint8_t*>(data), nbytes)
+        //           << std::endl;
+        return arrow::Status::OK();
+    }
+
+    arrow::Status Close() override { return arrow::Status::OK(); }
+
+    arrow::Result<int64_t> Tell() const override { return arrow::Result<int64_t>(0); }
+
+    bool closed() const override { return false; }
+
+private:
+    int _counter{0};
+};
+
+TEST(ArrowParquetWriterTest, basic_test) {
```

Comment:
no `EXPECT_EQ(xxx)`? Just to check this ut can run normally?

Reply:
Yes. UT would crash if arrow is not fixed by https://github.com/StarRocks/starrocks/pull/26253